job-dsls/jobs/osbs/bamoe_operators_osbs_seed_cloud_jobs.groovy

```diff
@@ -158,9 +158,25 @@ prodComponent.each { Component ->
                           checkout(githubscm.resolveRepository(operator_repo_name, GITHUB_ORG_UNIT, operator_branch, false))
                           // Run the build script that should be into the operator hack folder
                           dir('hack') {
-                            sh "source ~/virtenvs/cekit/bin/activate && ./${build_command}"
+                            sh "source ~/virtenvs/cekit/bin/activate && ./${build_command} | tee output.txt"
                           }
+                          
                           // post processing
+                          // query the built image from osbs using brew cli
+                          dir(component_path) {
```

Comment:
```suggestion
                          dir('hack') {
```

Reply:
another option could be to define a new variable `component_path` and set it equal to `hack` and use it

Reply:
it was a leftover, imho we can stick with dir(hack)